src/Raven.Server/Documents/Expiration/ExpiredDocumentsCleaner.cs

```diff
@@ -199,11 +199,11 @@ public void Dispose()
 
             if (SystemTime.UtcNow >= date)
                 throw new InvalidOperationException($"Cannot put an expired document. Expired on: {date.ToString("O")}");
-
-            var ticksBigEndian = Bits.SwapBytes((ulong)date.Ticks);
+            
+            var ticksBigEndian = IPAddress.HostToNetworkOrder(date.Ticks);
```

Comment:
Sorry, I overlooked somehow this comment. In general, when we we cast `long` values to `ulong` we cannot guarantee that they will results in the same ascending order (what we demand to properly iterate). For example:

            long a = -1;
            long b = 1;

            Debug.Assert(a < b);
            Debug.Assert(aUlong < bUlong); // FAIL
            ulong aUlong = (ulong) a;
            ulong bUlong = (ulong)b;


So, that's why I introduced that change. However, in this scenario `Ticks` are actually never negatives, so that shouldn't happen. If this happen again, I will get back to it.